src/foam/dao/history/HistoryRecordService.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.dao.history',
+  name: 'HistoryRecordService',
+
+  skeleton: true,
+
```

Comment:
```suggestion
client: true,
```
and then remove the HistoryRecordServiceClient as it will be auto generated.